src/TransportTests/TransportTests.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="AWSSDK.SQS" Version="3.3.3.10" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
     <PackageReference Include="NServiceBus.TransportTests.Sources" Version="7.1.0" />
-    <PackageReference Include="NUnit" Version="3.10.1" />
+    <PackageReference Include="NUnit" Version="[3.11.0, 4.0.0)" />
```

Comment:
Don't add version ranges on projects that don't produce a package.

Reply:
What @bording said